test/integration/test_scripts.py

```diff
@@ -185,6 +184,9 @@ def write_config_file(self):
         config_dir = self.config_dir
         path = os.path.join(config_dir, "galaxy.yml")
         self._test_driver.temp_directories.extend([config_dir])
+        config = self._raw_config
+        # Update config dict with database_connection, which might be set through env variables
+        config['database_connection'] = self._app.config.database_connection
         with open(path, "w") as f:
             yaml.dump({"galaxy": self._raw_config}, f)
```

Comment:
Should this be:
```suggestion
            yaml.dump({"galaxy": config}, f)
```
?

Reply:
It should, thanks!